static/app/views/replays/replayTable.tsx

```diff
@@ -1,148 +1,188 @@
-                  ? dataEntries[replay[idKey]]?.count_if_event_type_equals_error
-                  : 0}
-              </Item>
-              <Item>
-                <ReplayHighlight data={dataEntries[replay[idKey]]} />
-              </Item>
-            </React.Fragment>
-          ) : (
-            <React.Fragment>
-              <Item>
-                <Placeholder height="24px" />
-              </Item>
-              <Item>
-                <Placeholder height="24px" />
-              </Item>
-              <Item>
-                <Placeholder height="24px" />
-              </Item>
-            </React.Fragment>
-          )}
-        </Fragment>
+    <StyledPanelTable
+      isLoading={isFetching}
+      isEmpty={replays?.length === 0}
+      headers={[
+        t('Session'),
+        showProjectColumn && minWidthIsSmall ? t('Project') : null,
+        <SortLink
+          key="startedAt"
+          role="columnheader"
+          aria-sort={
+            sort.field === 'startedAt'
+              ? sort.kind === 'asc'
+                ? 'ascending'
+                : 'descending'
+              : 'none'
+          }
+          to={{
+            pathname,
+            query: {
+              ...location.query,
+              sort: sort.kind === 'desc' ? 'startedAt' : '-startedAt',
+            },
+          }}
+        >
+          {t('Start Time')} {sort.field.endsWith('startedAt') && sortArrow}
+        </SortLink>,
+        <SortLink
+          key="duration"
+          role="columnheader"
+          aria-sort={
+            sort.field.endsWith('duration')
+              ? sort.kind === 'asc'
+                ? 'ascending'
+                : 'descending'
+              : 'none'
+          }
+          to={{
+            pathname,
+            query: {
+              ...location.query,
+              sort: sort.kind === 'desc' ? 'duration' : '-duration',
+            },
+          }}
+        >
+          {t('Duration')} {sort.field === 'duration' && sortArrow}
+        </SortLink>,
+        t('Errors'),
+        t('Interest'),
+      ]}
+    >
+      {replays?.map(replay => (
+        <ReplayTableRow
+          key={replay.id}
+          replay={replay}
+          organization={organization}
+          showProjectColumn={showProjectColumn}
+          minWidthIsSmall={minWidthIsSmall}
+        />
       ))}
+    </StyledPanelTable>
+  );
+}
+
+function ReplayTableRow({
+  minWidthIsSmall,
+  organization,
+  replay,
+  showProjectColumn,
+}: RowProps) {
+  const {projects} = useProjects();
+  const project = projects.find(p => p.id === replay.projectId);
+  return (
+    <Fragment>
+      <UserBadge
+        avatarSize={32}
+        displayName={
+          <Link
+            to={`/organizations/${organization.slug}/replays/${project?.slug}:${replay.id}/`}
+          >
+            {replay.user.name ||
+              replay.user.email ||
+              replay.user.ip_address ||
+              replay.user.id ||
+              ''}
```

Comment:
I wonder if there's a way for the backend to re-use our existing user serializer. this way we could use `User.displayName`

Reply:
yeah, lets ticket it.